src/excmds.ts

```diff
@@ -2361,7 +2361,7 @@ if (fullscreenApiIsPrefixed) {
 
 /** @hidden */
 //#content
-export async function loadaucmds(cmdType: "DocStart" | "DocLoad" | "DocEnd" | "TabEnter" | "TabLeft" | "FullscreenEnter" | "FullscreenLeft" | "FullscreenChange" | "UriChange" | "HistoryState") {
+export async function loadaucmds(cmdType: "DocStart" | "DocLoad" | "DocEnd" | "TabEnter" | "TabLeft" | "FullscreenEnter" | "FullscreenLeft" | "FullscreenChange" | "UriChange" | "HistoryState" | "DownloadPost") {
     const aucmds = await config.getAsync("autocmds", cmdType)
     if (!aucmds) return
     const ausites = Object.keys(aucmds)
```

Comment:
Hey @antonva just casually looking at your 4 year-old PR

Your DownloadPost events are triggered based on the page currently being viewed

```
const aukeyarr = ausites.filter(e => window.document.location.href.search(e) >= 0)
```

Was that intentional? It seems a little weird to me